src/software/simulation/physics/physics_robot_model.h

```diff
@@ -17,11 +17,11 @@ class PhysicsRobotModel
      * Together these functions return 3 polygons that together make up the shape of the
      * robot. It is broken up this way because Box2D can only have convex polygons. We
      * split the body into 3 parts:
-     * - The main body, which is everything behind the chicker
-     * - The front-left part, which is the bit that is to the front-left of the chicker,
-     * partially enclosing it
-     * - The front-right part, which is the bit that is to the front-right of the chicker,
+     * - The main body, which is everything behind the dribbler
+     * - The front-left part, which is the bit that is to the front-left of the dribbler,
      * partially enclosing it
+     * - The front-right part, which is the bit that is to the front-right of the
```

Comment:
remove "that is" to make it more concise ("which is the bit to the front-right...")

Reply:
done